app/forms/steps/details/documents_checklist_form.rb

```diff
@@ -16,15 +19,20 @@ def persist!
       tribunal_case.update(
         original_notice_provided:             original_notice_provided,
         review_conclusion_provided:           review_conclusion_provided,
-        having_problems_uploading_documents:  having_problems_uploading_documents
+        having_problems_uploading_documents:  having_problems_uploading_documents,
+        having_problems_uploading_details:    having_problems_uploading_details
       )
     end
 
     def documents_uploaded
-      if tribunal_case && tribunal_case.documents.empty?
-        errors.add(:original_notice_provided, :no_documents)
-        errors.add(:review_conclusion_provided, :no_documents)
-      end
+      tribunal_case.documents.any? || errors.add(:original_notice_provided, :no_documents)
+    end
+
+    def letter_checkboxes
```

Comment:
`documents_uploaded` is a verb statement, might be good to have this as one as well so it fits in the validation.

How about `letter_checkboxes_ticked`?